tests/eas/fork_migration.config

```diff
@@ -0,0 +1,52 @@
+{
+    "modules"  : [ "bl" ],
+    "tools"    : [ "rt-app" ],
+    "ftrace" : { // TODO make these inherited
+        "events" : [
+            "sched_energy_diff",
+            "sched_load_avg_task",
+            "sched_load_avg_cpu",
+            "sched_migrate_task",
+            "sched_switch"
+        ],
+    },
+    "wloads" : {
+        // Create N 100% tasks and M 10% tasks which run in parallel, where N is
+        // the number of big CPUs and M is the number of LITTLE CPUs.
+        "fmig" : {
+            "type" : "rt-app",
+            "conf" : {
+                "class" : "profile",
+                "params" : {
+                    "small" : {
+                        "kind" : "Periodic",
+                        "params" : {
+                            "duty_cycle_pct": 10,
+                            "duration_s": 5,
+                            "period_ms": 10,
+                        },
+                        "tasks" : "little", // Create one task for each LITTLE CPU
```

Comment:
Agreed; I'll look at this after this rework is done.

I'm actually a little concerned that JSON representation might end up adding more complexity than simplicity because ideally we don't want to have any platform knowledge ("A53_0" or even "bigs"/"mediums") in the test. We can discuss this separately.